ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -1525,5 +1553,80 @@
     "FieldTypeParameter": "CaseOutsourcingType",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "additionalApplicationType",
+    "Label": "What are you applying for?",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "AdditionalApplicationType",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "usePbaPayment",
+    "Label": "Are you using PBA to pay?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2018",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "additionalApplicationsLabel",
+    "Label": "Use this feature to upload applications for additional orders. You might need to upload additional forms, depending on your application type.",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2018",
```

Comment:
are these PBA related fields are needed? there is complex type that keeps this information. I dont see access control configured for these fields so assume not used

Reply:
Removed pba fields